index.js

```diff
@@ -17,8 +17,24 @@ const EMAIL_PERMISSION = "alexa::profile:email:read";
 const EMPTY_STATE = 'empty';
 // used to move on to next step
 const TERMINATION_WORD = 'next';
+const helpWords = ['help'];
+const summaryWords = ['summary'];
+const restartWords = ['restart'];
 const MORE_INFO_PHRASE = 'Anything else?';
-const nextStepPhrases = ['nothing', 'nope', 'no', 'finish', 'next', 'continue', 'stop', 'that\'s it'];
+const nextStepPhrases = ['nothing', 'nope', 'no', 'finish', 'next', 'continue', 'stop', 'that\'s it', 'skip'];
+
+// TODO. make constants for questions.
+const QUESTION_PROJECT_NAME = "";
+const QUESTION_PLAN_SHORT = "";
+const QUESTION_PROBLEM = "";
+const QUESTION_LESSON = "";
+//...
+
+// Add a random hints at the beginning of start report
+const hints = ['Did you know you can say help at any time for a list of commands?',
```

Comment:
We want to make a function that randomly selects one of these hints and inserts it into the response we make when the skill is launched. This is for discoverability (proof of concept for now we can polish the implementation later)